app/assets/stylesheets/application.css

```diff
@@ -73,14 +73,32 @@ ul .inline {
   margin-top: 40px;
 }
 
-select#submission_event_id {
+select#submission_event_instance_id {
   display: block;
 }
 
 .g-recaptcha {
   margin-bottom: 1.5rem;
 }
 
+fieldset.fieldset {
+  border: 1px solid #c0c0c0;
+  border-radius: 3px;
+  margin: 0 2px;
+  margin-bottom: 15px;
+  padding: 0.5em 0.8em 0.75em;
+}
+
+/**
+ * 1. Correct `color` not being inherited in IE 8/9/10/11.
+ * 2. Remove padding so people aren't caught out if they zero out fieldsets.
```

Comment:
I don't understand what you're pointing out here. Can you explain please? Thank you. ☺️

Reply:
See above with thing about comment coming out of Normalize.